layers_ffjson.go

```diff
@@ -1,5 +1,5 @@
 // Code generated by ffjson <https://github.com/pquerna/ffjson>. DO NOT EDIT.
-// source: layers.go
+// source: ./layers.go
```

Comment:
This PR doesn't need this change.

Reply:
I think it does, but I will try without it.